### PR TITLE
fix: avoid reliance on Elm18Compatible in UserInteractions/Keyboard

### PR DESCRIPTION
**Pull request by mbylstra**
This upgrades some code in `UserInteractions/Keyboard.elm` to Elm 0.19 so that it no longer relies on a Elm 0.18 shim. This is part of a bigger effort to remove all usages of cultureamp/elm-upgrade-shims (https://github.com/cultureamp/kaizen-design-system/pull/577)

Ideally this would be using `event.key` (which is a string) instead of `event.keyCode` (which is an int), as that is the official recommendation for Elm 0.19, but that's out of scope for this PR.

### CODE REVIEW

File: draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm [hunk 1]
```diff
@@ -46,6 +47,17 @@ decoder keyCode =
...
+
 subscriptions : (Key -> msg) -> Sub msg
 subscriptions toMsg =
-    Keyboard18.downs (decoder >> toMsg)
+    Browser.Events.onKeyDown (keyDownDecoder |> Decode.map toMsg)
+
+
+{-| deprecated: this has been left in to avoid a breaking API change. This function is no longer used in this module. The name is confusing as it's not actually a decoder (it's just a transformation function)
+-}
+decoder =
```
Comment (Confidenceman02) [inline draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:62]: Could we type def this?

File: draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm [hunk 2]
```diff
@@ -46,6 +47,17 @@ decoder keyCode =
         Other
 
 
+keyDownDecoder : Decoder Key
+keyDownDecoder =
```
Comment (Confidenceman02) [inline draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:51]: We can use Html.Events.keyCode instead!

Comment (mbylstra) [reply draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:51]: sweet, didn't know that was there!

Comment (Confidenceman02) [reply draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:51]: There is even a mention about the deprecation of keyCode which is pretty neat.

Comment (mbylstra) [reply draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:51]: It should mention this in the docs for `Browser.Events.onKeyDown` though! (it tells you to write your own decoder)

File: draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm [hunk 3]
```diff
@@ -46,6 +48,18 @@ decoder keyCode =
...
+    Decode.map keyCodeToKey Html.Events.keyCode
```
Comment (Confidenceman02) [inline draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:53]: This is nit picky, could we expose `keyCode` please? :)

Comment (mbylstra) [reply draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:53]: This was intentional. I wanted to be clear that it's not something from this module.

Comment (Confidenceman02) [reply draft-packages/user-interactions/KaizenDraft/UserInteractions/Keyboard.elm:53]: Ah!!!! no probs!

**Pull request merged**